dist/scss/wds-components/_tabs.scss

```diff
@@ -29,7 +30,13 @@
 		transition: color .1s;
 
 		:not(:first-child):not(:only-child) {
```

Comment:
is `:not(:only-child)` needed? If some node isn't a first child it's rather obvious that it also is not an only child.